include/onnxruntime/core/framework/allocator.h

```diff
@@ -67,6 +126,8 @@ std::ostream& operator<<(std::ostream& out, const OrtAllocatorInfo& info);
 
 namespace onnxruntime {
 constexpr const char* CPU = "Cpu";
+constexpr const char* CUDA = "Cuda";
```

Comment:
Does it mean we may keep adding strings here when there's new device?

Reply:
I'm planning to retire the "string" id for a device, but an integer. It's not a must to have a device id/name added here.